.gitignore

```diff
@@ -142,6 +142,12 @@ cython_debug/
 
 # Include Spawn.log files required for tests
 !Spawn.log
+
+#Include figures for markdown
+!UED.png
+!P.png
+!R.png
+!Rblur.png
```

Comment:
Instead of having to add each individual image to our `.gitignore` (which future users will inevitably forget to do and will cause them endless headache as they struggle to figure out why their images aren't getting committed to the repo), let's make a `/images/` directory inside of `/docs` and keep all relevant images for the documentation there. Then you can add `!docs/images/*` to the `.gitignore` instead.